doc/nrf/ug_matter_gs_testing_wifi_mobile.rst

```diff
@@ -0,0 +1,43 @@
+.. _ug_matter_gs_testing_wifi_mobile:
+
+Matter over Wi-Fi: Configuring CHIP Tool for Android
+####################################################
+
+In this setup, the Matter controller is installed on mobile, and a dedicated Wi-Fi Access Point is used.
+
+.. figure:: images/matter_wifi_setup_mobile.png
+   :width: 600
+   :alt: Matter over Wi-Fi setup with Matter controller on mobile
+
+   Matter over Wi-Fi setup with Matter controller on mobile
+
+Requirements
+************
+
+To use this setup, you need the following hardware:
+
+* 1x smartphone with Android 8+
+* 1x Wi-Fi Access Point supporting IPv6 (without the IPv6 Router Advertisement Guard enabled on the router)
+* 1x compatible Nordic Semiconductor's DK - for the Matter accessory device (compatible and programmed with one of :ref:`matter_samples`)
+
+Configuring the environment
+***************************
+
+To configure and use CHIP Tool for Android, complete the following steps:
+
+1. Program the development kit for the Matter accessory device with one of available :ref:`matter_samples`.
+#. Configure the CHIP Tool for Android:
+
+   a. Install the controller using one of the options described in :ref:`ug_matter_configuring_controller_mobile`.
+   #. Complete the following steps from the :doc:`matter:nrfconnect_android_commissioning` user guide in the Matter documentation:
```

Comment:
Have we already updated `nrfconnect_android_commissioning.md` for Wi-Fi? If not, we may need another task to do so.

Reply:
We haven't, no.